src/components/tables/EditableTranscriptsTable.tsx

```diff
@@ -0,0 +1,227 @@
+import { useUserReviews } from "@/services/api/reviews";
+import { useClaimTranscript, useTranscripts } from "@/services/api/transcripts";
+import { calculateReadingTime, wordsFormat } from "@/utils";
+import { Box, Heading, Text, useToast } from "@chakra-ui/react";
+import axios from "axios";
+import { signIn, signOut, useSession } from "next-auth/react";
+import { useRouter } from "next/router";
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { Transcript } from "../../../types";
+import BaseTable from "./BaseTable";
+import { TableStructure } from "./types";
+
+const EmptyText = ({
+  hasActiveTranscript,
+  hasMorePendingTranscript,
+}: {
+  hasMorePendingTranscript: boolean;
+  hasActiveTranscript: boolean;
+}) => {
+  const props =
+    hasMorePendingTranscript || hasActiveTranscript
+      ? {
+          color: "black",
+          fontWeight: "bold",
+        }
+      : undefined;
+  const text = useMemo(() => {
+    if (hasActiveTranscript)
+      return "Please finish editing & submit the transcript you're working on before choosing a new one";
+    if (hasMorePendingTranscript)
+      return "You can only have 3 transcripts under review at once. Please wait until at least one is approved";
+
+    return "There are no transcripts that you can edit";
+  }, [hasActiveTranscript, hasMorePendingTranscript]);
+
+  return <Text {...props}>{text}</Text>;
+};
+
+const activeLimit = 1;
+const pendingLimit = 3;
+
+const EditableTranscriptsTable = () => {
+  const { data: session, status } = useSession();
+  const { data: activeReviews = [] } = useUserReviews({
```

Comment:
Instead of 2 api calls, We can reduce the number of api calls by fetching all user reviews by id (the data isn't much anyways). Then do a filter using the review status on the frontend.
We want to reduce bandwidth and multiple api calls